woocommerce/woocommerce-android
# Creating shipping label: Shipment cost section

Part of: #12281 

### Description
The changes in this PR include the shipment cost section UI implementation. All the changes in this PR are focused on UI, and the data used in the screens is mocked. On a different PR, I will connect this UI with real data.

### Testing information

1. Create an Order with the processing status on a store that can create shipping labels
2. Tap on Orders
3. Tap on the order created on step 1
4. Tap con create shipping labels
5. Check that Shipment details card is displayed at the bottom of the screen
6. Tap on/Drag up Shipment details
7. Check that the bottom sheets expand and the shipment cost section is visible

### The tests that have been performed
Test that the UI section is visible

### Images/gif
| First Header  | Second Header |
| ------------- | ------------- |
| <img width="300" src="https://github.com/user-attachments/assets/ba1eb74b-0f53-4205-8499-5496f725fd15" /> | <img width="300" src="https://github.com/user-attachments/assets/33141059-884c-4558-9670-ea6fc3d9b34e" />  |



## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
